userspace/libscap/scap_savefile.c

```diff
@@ -29,6 +29,7 @@ struct iovec {
 };
 #endif
 
+#include "../common/strlcpy.h"

```

Comment:
Yep :+1: , 99% happened because of a rebase. There was probably a `strncpy` down there that was since removed, while rebasing I removed the outdated lines but not the include. Will remove the commit for this file.